File: plusmall-operator-interface/src/main/java/com/plusmall/operator/ContentService.java
package com.plusmall.operator;

import com.plusmall.commons.PageResult;
import com.plusmall.model.TbContent;

/**
 * @Description:
 */
public interface ContentService {
	public PageResult search(int pageNum, int pageSize);
	public void delete(Long[] ids) throws NullPointerException;
	public void add(TbContent content) throws NullPointerException;
	public TbContent findOne(Long id);
	public void update(TbContent content) throws NullPointerException;
}

File: plusmall-operator-web/src/main/webapp/js/service/contentService.js
app.service('contentService',function ($http) {

    //查找
    this.search=function (pageNum, pageSize) {
        return $http.get('../content/search.do?pageNum='+pageNum+'&pageSize='+pageSize);
    }

    //删除
    this.delete=function (ids) {
        return $http.get('../content/delete.do?ids='+ids);
    }

    //添加
    this.add=function (content) {
        return $http.post('../content/add.do',content);
    }

    //根据id查找单个广告
    this.findOne=function (id) {
        return $http.get('../content/findOne.do?id='+id);
    }

    //更新广告
    this.update=function (content) {
        return $http.post('../content/update.do',content);
    }
})